config/routes.rb

```diff
@@ -15,6 +15,12 @@
     get :about
   end
 
+  resources :student, only: [] do
```

Comment:
Reminder that this is outdated. Will remove this in new PR that links students with their lessons